src/main/java/spoon/reflect/path/CtRole.java

```diff
@@ -59,25 +63,31 @@
 	ANNOTATION,
 	STATEMENT,
 	ARGUMENT,
-	SUPER_TYPE,
+	SUPER_TYPE("superclass"),
 	NESTED_TYPE,
 	CONSTRUCTOR,
 	EXECUTABLE,
-	FIELD,
-	CAST,
-	VALUE,
+	FIELD("typemembers"),
```

Comment:
executables and nested types are type members too ... it probably needs further refactoring ... but after tjis PR.